R/create_table.R

```diff
@@ -100,7 +100,9 @@ methods::setMethod("getTableSignature", "NULL", function(.data, conn) {
 #' @examples
 #' conn <- get_connection(drv = RSQLite::SQLite())
 #'
-#' create_logs_if_missing("test.logs", conn)
+#' if (schema_exists(conn, "test")) {
```

Comment:
Could you not just change the example a bit?

```
conn <- get_connection(drv = RSQLite::SQLite())

log_table_id <- SCDB::id("test.logs", conn)

create_logs_if_missing(log_table_id, conn)

close_connection(conn)
```

Doing so makes the example more readable (IMO)

Reply:
True! A consequence of committing too much at a time: You don't use the stuff you actually committed a bit earlier!

I have reverted the `schema_exists` check for this example.